include/random_walks/ellipsoid_walks/dikin_walker.h

```diff
@@ -0,0 +1,144 @@
+// Code from https://github.com/rzrsk/vaidya-walk
+
+// Modified by Alexandros Manochis, as part of Google Summer of Code 2020 program.
+
+#ifndef PWALK_DIKIN_WALKER_HPP_
+#define PWALK_DIKIN_WALKER_HPP_
+
+#include <Eigen/Dense>
+#include "math_functions.h"
+
+//namespace pwalk {
+
+template <typename Dtype>
+class DikinWalker {
+public:
+
+    DikinWalker() {}
+
+    //DikinWalker(const Eigen::Matrix<Dtype, Eigen::Dynamic, 1>& initialization, const Eigen::Matrix<Dtype, Eigen::Dynamic,
+    //        Eigen::Dynamic>& cons_A, const Eigen::Matrix<Dtype, Eigen::Dynamic, 1>& cons_b, const Dtype r) :
+    //        Walker<Dtype>(initialization, cons_A, cons_b), r_(r){}
+
+    void init(const Eigen::Matrix<Dtype, Eigen::Dynamic, 1> &initialization, const Eigen::Matrix <Dtype,
```

Comment:
why not at construction time ?

Reply:
Thanks. I have removed the init function.